api/capi/src/nnstreamer-capi-util.c

```diff
@@ -493,9 +493,10 @@ ml_tensors_data_destroy (ml_tensors_data_h data)
 
 /**
  * @brief Allocates a tensor data frame with the given tensors info. (more info in nnstreamer.h)
+ * @note Memory for data buffer is not allocated.
  */
 int
-ml_tensors_data_create (const ml_tensors_info_h info,
+ml_tensors_data_create_no_alloc (const ml_tensors_info_h info,
```

Comment:
What is the purpose of this line?

Reply:
This adds another tensor creation method, which does not allocate memory for the data inside it.
It just creates the tensor skeleton appropriately, and data tensor is left for the user to be allocated or set to already allocated memory in the current scenario.